api/bin/deal-observer-api.js

```diff
@@ -1,17 +1,15 @@
 import '../lib/instrument.js'
 import { createApp } from '../lib/app.js'
-import { createPgPool } from '@filecoin-station/deal-observer-db'
 
 const {
   PORT = '8080',
   HOST = '127.0.0.1',
-  REQUEST_LOGGING = 'true'
+  REQUEST_LOGGING = 'true',
+  DATABASE_URL = 'postgres://localhost:5432/spark_deal_observer'
```

Comment:
can we expose and read this default from https://github.com/filecoin-station/deal-observer/blob/8496edb6f84e0444993bf7f7fbffd9db6b28a4d4/db/index.js#L15 instead?